source/api_reference/index.html.md.erb

```diff
@@ -27,6 +27,5 @@ Using different endpoints, you can:
 * get a [specific item within a register](/api_reference/get_items_item_hash#get-items-item-hash)
 * download the [full contents of a register in a ZIP file](/api_reference/get_download_register#get-download-register)
 
-You can also read about [HTTP status codes](/api_reference/http_status_codes)
-and [rate limits](/api_reference/rate_limits).
+You can also read about [HTTP status codes](/api_reference/http_status_codes), [rate limits](/api_reference/rate_limits) and [version numbers](/api_reference/version-history.html).
```

Comment:
It might be a good idea to bullet these three items: it'll be easer and more accessible for users. 

Also, I'm not sure the `.html.md` extension is needed for the `version-history` file unless I'm missing some utility of it. It makes linking a little bit more complicated, as here.